client/blocks/subscription-length-picker/index.jsx

```diff
@@ -73,10 +72,10 @@ export class SubscriptionLengthPicker extends React.Component {
 							planSlug,
 							priceFull,
 							priceFullBeforeDiscount,
-							priceMinusCredits,
+							priceFinal,
 							priceMonthly,
 						} ) => {
-							const price = ! priceMinusCredits ? priceFull : priceMinusCredits;
+							const price = ! priceFinal ? priceFull : priceFinal;
```

Comment:
This is the same as `priceFinal || priceFull`. Just a personal preference, though :)